src/pages/About.vue

```diff
@@ -55,7 +55,7 @@ const { tracker } = useTracker();
       in the GitHub repository.
     </n-p>
     <n-p>
-      And if you found a bug, or something doesn't work as expected, please fill a bug report in the
+      And if you found a bug, or something doesn't work as expected, please file a bug report in the
```

Comment:
I add to double check this one, I was nearly certain that `fill` was correct, I thought it was a mistake in you previous PR, my bad
Thank you for re-fixing it 🙏🏻